scripts/microcleanfs.py

```diff
@@ -16,7 +16,10 @@
 """Remove the contents of the file system on a MicroPython device.
 
 Usage:
-    micrormdir PORT
+    micrormdir PORT [options]
```

Comment:
The name of the script should match the name of the file (`microcleanfs`) in order not to confuse people running it from the command line.

Reply:
I actually caught that in my docopt update and it is fixed in that branch.